src/stories/FormField.stories.tsx

```diff
@@ -39,3 +40,14 @@ VerticalFormField.args = {
     </>
   ),
 };
+
+export const VerticalFormField2 = Template.bind({});
```

Comment:
do you mind renaming the variable to reflect the flow? it's currently a VerticalForm with a `horizontal` flow lol

Reply:
Whoops good catch that was definitely a copy paste without a rename.